test/api/parameterizer.spec.js

```diff
@@ -0,0 +1,99 @@
+var assert = require('assert'),
+    sinon = require('sinon');
+
+var parameterizer = require('../../lib/dictionaries/api/parameterizer');
+
+
+
+describe('Parameterizer', function() {
+
+    var createMockResponse = function() {
+        return {
+            send: function() {},
+            status: function() {},
+            sendStatus: function() {}
+        }
+    };
+
+    it('should be a function', function() {
+        var p = parameterizer().default;
+        assert.equal('function', typeof p);
+    });
+
+    it('should invoke next when a users scope is passed', function() {
+
+        var req = {
+            params: {
+                scope: 'users'
+            }
+        };
+        var statusSet;
+        var res = {
+            sendStatus: function (status) {
+                statusSet = status;
+            }
+        };
+
+        var nextCalled =false;
+        function next() {
```

Comment:
Instead of doing it manually playing with the scope you could use a spy.
```javascript
var next = sinon.spy();
 parameterizer().default(req, res, next);
assert(next.calledOnce);
```